src/main/java/org/opensearch/knn/plugin/transport/UpdateModelGraveyardTransportAction.java

```diff
@@ -156,4 +168,72 @@ public ClusterTasksResult<UpdateModelGraveyardTask> execute(ClusterState cluster
             return new ClusterTasksResult.Builder<UpdateModelGraveyardTask>().successes(taskList).build(updatedClusterState);
         }
     }
+
+    private static void checkIfIndicesAreUsingModel(ClusterState clusterState, UpdateModelGraveyardTask task, String[] indicesArray)
```

Comment:
can we just make private method? Also, why use an array instead of list for indices?

Reply:
findMappings() later in the class takes a string array as a parameter.